src/components/views/settings/index.tsx

```diff
@@ -163,7 +178,21 @@ export const Settings: React.FC<SettingsProps> = ({navigation}) => {
                   accessibilityLabel={label}
                   accessibilityRole="button"
                   accessibilityHint={hint}
-                  onPress={() => navigation.navigate(screen)}>
+                  onPress={() =>
+                    screen === 'test'
+                      ? Linking.openURL(
+                          `mailto:support@example.com?subject=App Feedback for COVID Alert NY: Version ${version}&body=\n\n\n\n
+                          Device: ${getModel()}
```

Comment:
Missing app version above device. Hint: it's already defined as a variable on this screen.

Reply:
Also, if we need newline characters we should also include them at the end of these extra lines too

Reply:
There is a `RNDeviceInfo` that we just added that we can use (if you didn't already... :P)